spec/compute_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+require_relative './shared_context'
+
+require 'fog/kubevirt/compute/compute'
+
+describe Fog::Kubevirt::Compute do
+  before :all do
+    vcr = KubevirtVCR.new(
+      :vcr_directory => 'spec/fixtures/kubevirt/service',
+      :service_class => Fog::Kubevirt::Compute
+    )
+    @service = vcr.service
+  end
+
+  it 'CRUD services' do
+	VCR.use_cassette('services_crud') do
+      begin
+        port = 30000
+        vmi_name = "vmi-fedora"
+        srv_name = "#{vmi_name}-ssh"
+        @service.services.create(port: port, name: srv_name, target_port: 22, vmi_name: vmi_name)
+
+        srv = @service.services.get(srv_name)
```

Comment:
no exceptions for successful creation with the expected values ?

Reply:
No, it is only successful flow.